README.md

```diff
@@ -1 +1,128 @@
 # environment-variable-policy
+
+The environment-variable-policy can be used to inspect environment variables
+defined in the resources deployed in the cluster. It's able to validate both
+variables names and values. The policy allows the users define multiple validation rules.
+And the resource must pass **all** the rules to be allowed in the cluster.
+
+The policy can either target Pods, or workload resources (Deployments, ReplicaSets,
+DaemonSets, ReplicationControllers, Jobs, CronJobs). Both have trade-offs:
+
+- Policy targets Pods: Different kind of resources (be them native or CRDs) can
+  create Pods. By having the policy target Pods, we guarantee that all the Pods
+  are going to be compliant, even those created from CRDs. However, this could
+  lead to confusion among users, as high level Kubernetes resources would be
+  successfully created, but they would stay in a non reconciled state.
+  Example: a Deployment creating a non-compliant Pod would be created, but it
+  would never have all its replicas running.
+
+- Policy targets workload resources (e.g: Deployment): the policy inspect higher
+  order resource (e.g. Deployment): users will get immediate feedback about rejections.
+  However, non compliant pods created by another high level resource (be it native
+  to Kubernetes, or a CRD), may not get rejected.
+
+## Settings
+
+Each rule defined in the policy settings is composed by a set operator and set
+of the environment variable used with the operator against the environment variables
+from the resources. The rules are evaluated in the order that they are defined.
+The resource is denied in the first failed evaluated rule. The following yaml is a settings example:
+
+```yaml
+settings:
```

Comment:
please update the settings with the latest changes. It should be `reject` instead of `operator`
something like
```
reject: anyIn
environmentVariables:
   - name: "envvar1"
     value: "envvar1_value"
```
copy this changes to the metadata.yml too